java/client/src/main/java/glide/api/models/configuration/AdvancedBaseClientConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.configuration;
+
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Advanced configuration settings class for creating a client. Shared settings for standalone and
+ * cluster clients.
+ */
+@Getter
+@SuperBuilder
+public abstract class AdvancedBaseClientConfiguration {
+
+    /**
+     * The duration in milliseconds to wait for a TCP/TLS connection to complete. This applies both
```

Comment:
Maybe you need to update doc for all clients:

"connection to complete" -> "connections to all server nodes to complete"

Is that true BTW? It is not tested - consider modifying a test to run `DEBUG` to a random node.

Reply:
if debug runs on a random node, than I will be able to establish connection

Reply:
so my guess is that its not true

Reply:
In that case we're going to https://github.com/valkey-io/valkey-glide/issues/850#issuecomment-1910718990.
If it is not true we may get in a situation where client connected to only one node.
@ikolomi what do you think?